app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="categories">
+
+
+<div class="categories-list">
+  <% @categories.includes([:user]).order('created_at DESC').each do |category| %>
+    <% if can? :manage, category %>
+        <%= render category %>
+    <% end %>
+  <% end %>
+</div>
+
+<button class="newCatBtn">
+<%= link_to "New Category", new_category_path, class: 'newCatbtntext' %>
+</button>
+
```

Comment:
- [ ] As suggested by the previous reviewer, please try to apply the correct styles to the following button in order to match the one on the design given 😃 I tried to add a comparison so that you can get a sense of what you need to add 😉 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/209003649-9a843be7-8979-4f29-b14f-d8aa018e85ad.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/209003727-b3ccf73d-db33-4e4d-8cb0-dc16adcd6620.png)

As a hint, make sure it fills the whole width of the screen and the texts are aligned in the center and other things you can notice there 😉